scala-xml/src/test/scala/scalaxml/XmlGenerators.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Derived from https://raw.githubusercontent.com/arktekk/anti-xml/973068681c74e22bc8c68f4c116720ed543cc0e3/src/test/scala/com/codecommit/antixml/XMLGenerators.scala
```

Comment:
The original anti-xml version is nicer and more complete.  This is a hackjob to get us some additional coverage from the only scalacheck-XML integration I could find.